card/lib/cardio.rb

```diff
@@ -76,8 +77,22 @@ def set_paths paths
       add_path "db/seeds", :with => "db/seeds.rb"
 
       add_path 'config/initializers',  :glob => '**/*.rb'
-      paths['config/initializers'] << "#{gem_root}/mod/**{,/*/**}/initializers"
-      paths['config/initializers'] << "#{root}/mod/**{,/*/**}/initializers"
+
+    end
+
```

Comment:
Ok, I think I get everything except the 'override said paths' part.

So, it works for mods in the deck under mod/.../initializers/, but sometimes you want to change that?  Is this something you need to do now for deck modules and have to work-around, or?